app/controllers/report_controller.rb

```diff
@@ -58,7 +45,37 @@ def add_record
   end
 
   def raw_data
-    render json: Report.all
+    reports = Report.order(:max)
+    response = {}.tap do |res|
+      res[:max_main] = reports.last.max
+      res[:header] = {}.tap do |h|
+        h[:user] = 'User'
+        h[:platform] = 'Platform'
+        h[:location] = 'Location'
+        h[:core_number] = 'NO. cores'
+        h[:ram] = 'RAM(GB)'
+        h[:main] = 'Carbon emissions per core at full load (kgCO2eq/h)'
```

Comment:
I think this looks a little wordy on the page. I think it could be made easier to read, e.g.  if it was laid out like this... 
![image](https://github.com/openflighthpc/carbon-leaderboard/assets/102584263/4a6f16bf-0e01-4e22-929c-2ffe987de725)

here's the html if you'd like it 
```
      CO<sub>2</sub> emissions <br>
      <span class="small-text">per core at full load</span><br>
      (kg/h)
```